<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrainSuiteInstallDialog</class>
 <widget class="QDialog" name="BrainSuiteInstallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrainSuite14 Installer</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BrainSuite Installer</string>
   </property>
  </widget>
  <widget class="QLabel" name="explanationInfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>460</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>It appears that this is the first time BrainSuite15 has been run on this account. In order for certain features to work properly, BrainSuite14 needs to store some files on your local file system. The typical location for this would be in a subdirectory in your home directory.
If you choose not to install at this time, you will be prompted again the next time you run BrainSuite15.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="proceedCommandLinkButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceed without installing these files</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="installCommandLinkButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Install these files to the specified location</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="installDirectoryLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>\</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="installDirectoryLocationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Install Location:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="installDirectoryInfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>will be installed</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="installLogTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>460</width>
     <height>131</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="selectPushButton">
      <property name="text">
       <string>Select Installation Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetToDefaultLocationPushButton">
      <property name="text">
       <string>Reset to Default Location</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="allowInternetConnectionCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this box is checked, each time BrainSuite is started it will check its server to see if there are updates. This can be enabled or disabled via the help menu. If enabled, updates are also checked when About BrainSuite menu item is selected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Allow BrainSuite to connect to the Internet to check for updates.</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
